doc/source/whatsnew/v1.5.0.rst

```diff
@@ -206,6 +206,7 @@ Performance improvements
 - Performance improvement in :meth:`DataFrame.duplicated` when subset consists of only one column (:issue:`45236`)
 - Performance improvement in :meth:`.GroupBy.transform` when broadcasting values for user-defined functions (:issue:`45708`)
 - Performance improvement in :meth:`.GroupBy.transform` for user-defined functions when only a single group exists (:issue:`44977`)
+- Performance improvement in :meth:`DataFrame.merge` when left and/or right are empty (:issue:`45838`)
```

Comment:
I func we normally use :func: merge instead of DataFrame.merge?

Reply:
updated to :func: merge. thanks